utcp.c

```diff
@@ -50,6 +51,10 @@
 } while (0)
 #endif
 
+#ifndef max
+#define max(a, b) ((a) > (b) ? (a) : (b))
```

Comment:
This would call operations twice. For example this would have unexpected results: `max( i++, 10 )` because `i++` would be called twice. It's much safer and no problem performance-wice to make this a small inline function.